src/Webapi/Dom/Webapi__Dom__HtmlMediaElement.re

```diff
@@ -0,0 +1,139 @@
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement
+ */
+module Impl = (T: {type t;}) => {
+  type t_htmlMediaElement = T.t;
+
+  type mediaProvider; // TODO: one of MediaStream, MediaSource, or Blob
```

Comment:
Thinking this might be a `Webapi__MediaProvider.t` instead of a local phantom type